reviewrot/githubstack.py

```diff
@@ -0,0 +1,167 @@
+import logging
+import datetime
+from dateutil.relativedelta import relativedelta
+from base import BaseService, BaseReviewRot
+from github import Github
+from github.GithubException import UnknownObjectException
+
+log = logging.getLogger(__name__)
+
+
+class GithubService(BaseService):
+    """
+    This class represents Github. The reference can be found here:
+    https://developer.github.com/v3/
+    """
+    def request_reviews(self, user_name, repo_name=None, state_=None,
+                        value=None, duration=None, token=None, host=None):
+        """
+        Creates a github object.
+        Requests pull requests for specified username and repo name.
+        If repo name is not provided then requests pull requests
+        for all repos for specified username/organization.
+
+        Args:
+            user_name (str): Github username or organization name
+            repo_name (str): Github repository name for specified
+                             username or organization
+            state_ (str): The filter state for pull requests, e.g, older
+                        or newer
+            value (int): The value in terms of duration for requests
+                      to be older or newer than
+            duration (str): The duration in terms of period(year, month,
+                         hour, minute) for requests to be older or newer than
+            token (str): Github token for authentication
+            host (str): Github host name (Supposed to be None for project
```

Comment:
You might want to document that this value is not yet supported and the default behavior is to use the public github instance. It is possible to install an enterprise version of github. Perhaps a future enhancement could use this argument.